core/dbt/contracts/graph/model_config.py

```diff
@@ -285,7 +286,7 @@ def same_contents(cls, unrendered: Dict[str, Any], other: Dict[str, Any]) -> boo
     # 'meta' moved here from node
     mergebehavior = {
         "append": ["pre-hook", "pre_hook", "post-hook", "post_hook", "tags"],
-        "update": ["quoting", "column_types", "meta"],
+        "update": ["quoting", "column_types", "meta", "docs"],
         "dict_key_append": ["grants"],
```

Comment:
Did you get an error or did it just not add values under `model_id.config.docs`?

I feel like "update" is what we want to do. If a model has `docs` defined because of the folder it is in, if we redefine it at the model level we just want the most specific value to be kept (rather than combining with the folder one).

Reply:
I got it working at the folder level!

However, it overrides the `schema.yml` configs when it should be the other way around.

![image](https://user-images.githubusercontent.com/19176976/174805231-c9675d9f-a10f-4d2f-852e-043a1f57e886.png)
![image](https://user-images.githubusercontent.com/19176976/174805263-964161b1-58cb-41b6-850c-a035510bc4af.png)
![image](https://user-images.githubusercontent.com/19176976/174805290-ddc3c62c-86eb-4382-a04b-3fea3ada5228.png)

Reply:
Ahhh yes, this must be it. Thanks for writing this out explicitly. I'm aligned with your approach!